pkg/provider/azure.go

```diff
@@ -768,19 +734,14 @@ func (az *Cloud) configAzureClients(
 	// Initialize all azure clients based on client config
 	az.InterfacesClient = interfaceclient.New(interfaceClientConfig)
 	az.VirtualMachineSizesClient = vmsizeclient.New(vmSizeClientConfig)
-	az.StorageAccountClient = storageaccountclient.New(storageAccountClientConfig)
 	az.DisksClient = diskclient.New(diskClientConfig)
 	az.VirtualMachinesClient = vmclient.New(vmClientConfig)
 	az.VirtualMachineScaleSetsClient = vmssclient.New(vmssClientConfig)
 	az.VirtualMachineScaleSetVMsClient = vmssvmclient.New(vmssVMClientConfig)
 	az.SubnetsClient = subnetclient.New(subnetClientConfig)
 	az.LoadBalancerClient = loadbalancerclient.New(loadBalancerClientConfig)
 	az.PublicIPAddressesClient = publicipclient.New(publicIPClientConfig)
-	az.FileClient = fileclient.New(fileClientConfig)
```

Comment:
@MartinForReal could you add back file, blob clients in Azure.go? we need accountRepo and other repo initilized in azure.go


```
root@andydev:~/go/src/sigs.k8s.io/azurefile-csi-driver/pkg/azurefile# go test
# sigs.k8s.io/azurefile-csi-driver/pkg/azurefile [sigs.k8s.io/azurefile-csi-driver/pkg/azurefile.test]
./azurefile.go:826:30: d.cloud.GetStorageAccesskeyFromServiceAccountToken undefined (type *provider.Cloud has no field or method GetStorageAccesskeyFromServiceAccountToken)
./azurefile.go:853:45: d.cloud.StorageAccountClient undefined (type *provider.Cloud has no field or method StorageAccountClient)
./azurefile.go:855:33: d.cloud.GetStorageAccesskey undefined (type *provider.Cloud has no field or method GetStorageAccesskey)
./azurefile.go:986:18: d.cloud.FileClient undefined (type *provider.Cloud has no field or method FileClient)
./azurefile.go:1027:18: d.cloud.FileClient undefined (type *provider.Cloud has no field or method FileClient)
./azurefile.go:1079:29: d.cloud.FileClient undefined (type *provider.Cloud has no field or method FileClient)
./azurefile.go:1094:31: d.cloud.StorageAccountClient undefined (type *provider.Cloud has no field or method StorageAccountClient)
./azurefile.go:1109:21: d.cloud.RemoveStorageAccountTag undefined (type *provider.Cloud has no field or method RemoveStorageAccountTag)
./azurefile.go:1142:29: d.cloud.GetStorageAccesskey undefined (type *provider.Cloud has no field or method GetStorageAccesskey)
./controllerserver.go:386:37: d.cloud.StorageAccountClient undefined (type *provider.Cloud has no field or method StorageAccountClient)
./controllerserver.go:386:37: too many errors
FAIL    sigs.k8s.io/azurefile-csi-driver/pkg/azurefile [build failed]
```